lib/main_common.pm

```diff
@@ -366,6 +366,8 @@ sub load_yast2_ui_tests {
     if (check_var('DISTRI', 'opensuse') && !get_var("LIVETEST") && !get_var("LAPTOP")) {
```

Comment:
I think this will be more clear way to do same thing 

```
if (!get_var("LIVETEST") && !get_var("LAPTOP")) {
       if (check_var('DISTRI', 'opensuse')) {
          loadtest "console/yast2_cmdline";
          loadtest "console/yast2_dns_server";
}
          loadtest "console/yast2_nfs_client";
      }
```